dashboard/config/course_offerings/20-hour.json

```diff
@@ -0,0 +1,4 @@
+{
+  "key": "20-hour",
+  "display_name": "20-hour"
+}
```

Comment:
please include a newline at the end of these files, in the code that is generating these files